modules/porous_flow/tests/poro_elasticity/terzaghi_fully_saturated_volume.i

```diff
@@ -0,0 +1,361 @@
+    porepressure = porepressure
+  [../]
+  [./massfrac]
+    type = PorousFlowMassFraction
+  [../]
+  [./dens0_qp]
+    type = PorousFlowDensityConstBulk
+    density_P0 = 1
+    bulk_modulus = 8
+    phase = 0
+  [../]
+  [./dens_all_at_quadpoints]
+    type = PorousFlowJoiner
+    material_property = PorousFlow_fluid_phase_density_qp
+    at_nodes = false
+  [../]
+  [./porosity]
+    type = PorousFlowPorosityConst # only the initial value of this is used
+    porosity = 0.1
+  [../]
+  [./biot_modulus]
+    type = PorousFlowConstantBiotModulus
+    biot_coefficient = 0.6
+    fluid_bulk_modulus = 8
+    solid_bulk_compliance = 0.25
+  [../]
+  [./permeability]
+    type = PorousFlowPermeabilityConst
+    permeability = '1.5 0 0   0 1.5 0   0 0 1.5'
+  [../]
+  [./visc0]
+    type = PorousFlowViscosityConst
+    viscosity = 0.96
+    phase = 0
+  [../]
+  [./visc_all]
+    type = PorousFlowJoiner
+    material_property = PorousFlow_viscosity_qp
+  [../]
+[]
+
+[Postprocessors]
+  [./p0]
+    type = PointValue
+    outputs = csv
+    point = '0 0 0'
+    variable = porepressure
+    use_displaced_mesh = false
+  [../]
+  [./p1]
+    type = PointValue
+    outputs = csv
+    point = '0 0 1'
+    variable = porepressure
+    use_displaced_mesh = false
+  [../]
+  [./p2]
+    type = PointValue
+    outputs = csv
+    point = '0 0 2'
+    variable = porepressure
+    use_displaced_mesh = false
+  [../]
+  [./p3]
+    type = PointValue
+    outputs = csv
+    point = '0 0 3'
+    variable = porepressure
+    use_displaced_mesh = false
+  [../]
+  [./p4]
+    type = PointValue
+    outputs = csv
+    point = '0 0 4'
+    variable = porepressure
+    use_displaced_mesh = false
+  [../]
+  [./p5]
+    type = PointValue
+    outputs = csv
+    point = '0 0 5'
+    variable = porepressure
+    use_displaced_mesh = false
+  [../]
+  [./p6]
+    type = PointValue
+    outputs = csv
+    point = '0 0 6'
+    variable = porepressure
+    use_displaced_mesh = false
+  [../]
+  [./p7]
+    type = PointValue
+    outputs = csv
+    point = '0 0 7'
+    variable = porepressure
+    use_displaced_mesh = false
+  [../]
+  [./p8]
+    type = PointValue
+    outputs = csv
+    point = '0 0 8'
+    variable = porepressure
+    use_displaced_mesh = false
+  [../]
+  [./p9]
+    type = PointValue
+    outputs = csv
+    point = '0 0 9'
+    variable = porepressure
+    use_displaced_mesh = false
+  [../]
+  [./p99]
+    type = PointValue
+    outputs = csv
+    point = '0 0 10'
+    variable = porepressure
+    use_displaced_mesh = false
+  [../]
+  [./zdisp]
+    type = PointValue
+    outputs = csv
+    point = '0 0 10'
+    variable = disp_z
+    use_displaced_mesh = false
+  [../]
+  [./dt]
+    type = FunctionValuePostprocessor
+    outputs = console
+    function = if(0.5*t<0.1,0.5*t,0.1)
+  [../]
+[]
+
+
+[Preconditioning]
+  [./andy]
+    type = SMP
+    full = true
+    petsc_options_iname = '-ksp_type -pc_type -snes_atol -snes_rtol -snes_max_it'
```

Comment:
Curious, why do you set the snes tolerances through PETSc options instead of using the parameters in the executioner block? I see this pattern elsewhere in your code too.